sources/Interop/Kernel32/um/synchapi/Kernel32.Manual.cs

```diff
@@ -17,8 +17,6 @@ public static unsafe partial class Kernel32
 
         public const uint INIT_ONCE_CTX_RESERVED_BITS = RTL_RUN_ONCE_CTX_RESERVED_BITS;
 
-        public const uint CONDITION_VARIABLE_INIT = RTL_CONDITION_VARIABLE_INIT;
```

Comment:
This was lost/dropped?

Reply:
I removed it because the actual definition in `winnt.h` is this:
```c
#define RTL_CONDITION_VARIABLE_INIT {0}
```
Whereas this defines it as a `uint`. It seems to me there is no translation for what `{0}` does in C and that defining this as a number whatsoever is incorrect. I decided to be bold.